pkg/web/templates/partials/sunrise/sunrise_reject.html

```diff
@@ -0,0 +1,15 @@
+<div class="p-4 my-4 border bg-[#FFDDDD]" hx-target="#sunrise-response" hx-swap="outerHTML">
+  
+  <div class="flex items-baseline gap-x-2">
+    <i class="fa-solid fa-circle-check text-lg text-success"></i>
+    <p class="py-2 text-lg">
+      The secure message transaction has been rejected. A confirmation message will be sent to {originator} with the rejected reason of {reject_reason}. Click the <span class="font-semibold">download</span> button below to save a copy of the transaction data for your records.
```

Comment:
This will be handled similar to reject; however, once a user clicks the reject button in the form a rejection modal will appear. Once the user selects a rejection reason, then this partial will be loaded into the element with the `sunrise-response` id.